src/adapters/sonobi.js

```diff
@@ -0,0 +1,76 @@
+var bidfactory = require('../bidfactory.js');
+var bidmanager = require('../bidmanager.js');
+var adloader = require('../adloader.js');
+var utils = require('../utils');
+
+var SonobiAdapter = function SonobiAdapter(){
+  var test = false;               //  tag tester = true || false
+  var cb_map = {};
+
+  function _phone_in(params){
+    var trinity = '//apex.go.sonobi.com/trinity.js?key_maker=';
+    var bids = params.bids || [];
+    adloader.loadScript(trinity + JSON.stringify(_keymaker(bids)) + '&cv=' + _operator(), null);
+  }
+
+  function _keymaker(bids){       //  Make keys
+    var keyring = {};
+    bids.forEach(function(o){
```

Comment:
please use `utils._each(bids, function() { ... })` as we target support to IE8+.